internal/serverinstall/ecs.go

```diff
@@ -1063,6 +1073,13 @@ func (i *ECSInstaller) InstallFlags(set *flag.Set) {
 		Usage:   "Configures the requested memory amount for the Waypoint server task in ECS.",
 		Default: "1024",
 	})
+
+	set.BoolVar(&flag.BoolVar{
+		Name:    "ecs-alb-internal",
```

Comment:
Nit: At this time, it's technically a [Network Load Balancer](https://aws.amazon.com/elasticloadbalancing/features/#Product_comparisons) . Ideally it would be an ALB but there's issues with using gRPC and ALBs at the moment that we have not solved, but I'm hopeful in the future we could. 

For now I think we should rename this to `ecs-lb-internal` and just refer to it as "load balancer", dropping the specifics of Application or Network. Thoughts?